023-quarkus-kamelet/README.md

```diff
@@ -0,0 +1,16 @@
+# Quarkus - Kamelet
+
+The aim of this module is to cover all kamelet scenarios. 
```

Comment:
"all" is too vague, please list the scenarios and highlight the one hit by Camel Quarkus team